packages/terra-profile-image/src/ProfileImage.jsx

```diff
@@ -16,6 +16,10 @@ const propTypes = {
    * The text content that specifies an alternative text for an image.
    */
   alt: PropTypes.string,
+  /**
+  * Sets the `fit` prop of terra-image from the following values: `cover`, `contain`, `fill`, `scale-down`, `none`.
```

Comment:
`terra-image` is internal implementation of this component. I'd not mention it in the documentation.

Reply:
I agree. Addressed it here : https://github.com/cerner/terra-core/pull/2804/commits/c0cbab974a3b0b0a47de3ffd04a401a78dfb3265